src/core/less/global/fonts.less

```diff
@@ -0,0 +1,139 @@
+// Font setup
+// ==========
+
+html {
+  -ms-text-size-adjust: 100%; // preserve HTML font-size (iPhone orientation change)
+  -webkit-text-size-adjust: 100%;
+} 
+
+body {
+    color: @font-color;
+    font-family: @font-family;
+    font-size: @body-text-font-size;
+    font-weight: @body-text-font-weight;
+    line-height: @body-text-line-height;
+    -webkit-font-smoothing: antialiased;
+}
+
+.ie8 & {
+    body {
+        font-family: Arial, sans-serif;
+    }
+}
+
+// Variable set font styling
+// =========================
+
+.menu-title {
```

Comment:
can we remove .menu-title > .button text and all references to these mixins?

they are superfluous to the core theme stuffs and will probably need to be overridden in a theme